core/arch/arm/plat-imx/conf.mk

```diff
@@ -1,11 +1,20 @@
 PLATFORM_FLAVOR ?= mx6ulevk
 
-ifeq ($(PLATFORM_FLAVOR),mx6ulevk)
-arm32-platform-cpuarch		:= cortex-a7
+# Get SoC associated with the PLATFORM_FLAVOR
+ifeq ($(PLATFORM_FLAVOR),$(filter $(PLATFORM_FLAVOR),mx6ulevk))
+$(call force,CFG_MX6UL,y)
 endif
-ifeq ($(PLATFORM_FLAVOR),$(filter $(PLATFORM_FLAVOR),mx6qsabrelite mx6qsabresd mx6dlsabresd))
-arm32-platform-cpuarch		:= cortex-a9
+
+ifeq ($(PLATFORM_FLAVOR),$(filter $(PLATFORM_FLAVOR),mx6qsabrelite mx6qsabresd))
```

Comment:
`ifneq (,$(filter $(PLATFORM_FLAVOR),mx6qsabrelite mx6qsabresd))` would be simpler